package.json

```diff
@@ -4,13 +4,14 @@
   "description": "A Typescript library to help you get an API server up and running with I/O schema validation and custom middlewares in minutes.",
   "license": "MIT",
   "scripts": {
-    "start": "ts-node example/index.ts",
+    "esm": "NODE_OPTIONS=\"--loader ts-node/esm --es-module-specifier-resolution=node\" node",
```

Comment:
need to change all imports to complete filenames in order to get rid of this option
`--es-module-specifier-resolution`